controllers/chat/deleteMessage.js

```diff
@@ -18,27 +18,27 @@ const deleteMessage = async (req, res) => {
     await client.partialUpdateMessage(messageID, {
       set: {
         message_type: MESSAGE_TYPE.DELETED,
-        text: 'This message has been deleted'
+        text: 'This message has been deleted',
+        attachments: []
       }
     });
-    let destroy = await client.deleteMessage(messageID);
+
     // sent empty message to channel
     let baseMessage = {
       user_id: req.userId,
-      message_type: 'deleted',
+      message_type: 'notification-deleted',
+      text: 'This message has been deleted',
       deleted_message_id: messageID,
-      deleted_message_created_at: message?.created_at,
-      deleted_message_updated_at: message?.updated_at
+      deleted_message_created_at: message?.message?.created_at,
+      deleted_message_updated_at: message?.message?.updated_at
     };
+
+    console.log('baseMessage', baseMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging the entire `baseMessage` object directly can potentially expose sensitive data in your logs. Be careful with what you log, especially in production environments. If you need to log this for debugging purposes, consider logging only the necessary fields or obfuscating sensitive data.



<!-- This is an auto-generated comment by OSS CodeRabbit -->